src/components/Home/Home.jsx

```diff
@@ -0,0 +1,27 @@
+import { useUser } from '../../context/UserContext';
+import { Link } from 'react-router-dom';
+
+export default function Home() {
+  const { user } = useUser();
+
+  return (
+    <>
+      <h1>Welcome to the Monsters Inc. Employee Directory!</h1>
+      <p>
+        As an employee of Monsters Inc., you're required to create a profile on
+        our website in order to gain access to stuff.
+      </p>
+      <p>Create an account, then log in and fill out your account details.</p>
+      <p>Thank you for your participation.</p>
+      {user?.email ? (
```

Comment:
`user` always exists as an object so the optional chaining here probably isn't necessary